components/updates/UpdateAudienceBreakdown.js

```diff
@@ -33,7 +33,7 @@ const UpdateAudienceBreakdown = ({ audienceStats, isLoading }) => {
   if (isLoading) {
     return <LoadingPlaceholder height={50} />;
   } else if (!audienceStats) {
```

Comment:
We'll need to update the condition here if we apply https://github.com/opencollective/opencollective-api/pull/7004#discussion_r779520347

Reply:
Updated 👍🏻